server/src/main/scala/services/graphstages/StaffingStage.scala

```diff
@@ -4,18 +4,28 @@ import java.util.UUID
 
 import akka.stream._
 import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
-import drt.shared.CrunchApi.{CrunchMinute, MillisSinceEpoch, PortState, StaffMinute}
+import drt.shared.CrunchApi._
 import drt.shared.FlightsApi.{QueueName, TerminalName}
 import drt.shared.{MilliDate, Queues, SDateLike, StaffMovement}
 import org.slf4j.{Logger, LoggerFactory}
-import services.graphstages.Crunch.desksForHourOfDayInUKLocalTime
+import services.graphstages.Crunch.{desksForHourOfDayInUKLocalTime, getLocalLastMidnight, getLocalNextMidnight}
 import services.graphstages.StaffDeploymentCalculator.{addDeployments, queueRecsToDeployments}
 import services.{OptimizerConfig, SDate, TryRenjin}
 
+import scala.collection.JavaConverters._
+//import scala.collection.immutable.Map
```

Comment:
it this left in by mistake or as a warning?

Reply:
mistake i think!